photo-app/app/assets/javascripts/credit_card_form.js

```diff
@@ -0,0 +1,58 @@
+let show_error, submitHandler;
+
+show_error = function (message) {
+  console.log(message);
+  if(!$("#flash-messages").length){
+    $('#main').prepend("<div id='flash-messages'></div>");
+  }
+
+  $("#flash-messages").html('<div class="alert alert-warning"><a class="close" data-dismiss="alert">×</a><div id="flash_alert">' + message + '</div></div>');
+  $('.alert').delay(5000).fadeOut(3000);
+
+  return false;
+};
+
+function stripeTokenHandler(token, form){
+  form.append($("<input type=\"hidden\" name=\"payment[token]\" />").val(token.id));
+  form.get(0).submit();
+};
+
+function createToken(form){
+  stripe.createToken(card).then(function(result) {
```

Comment:
where did you get `stripe` from?

Reply:
The documentation. The stripe JS has been included with the CDN.
![image](https://user-images.githubusercontent.com/67599130/165903227-fa8e1bce-d412-4132-80ac-5f74d45dae7a.png)

Reply:
Bad way of writing. You basically add it to the global scope which polute it and also its in a place where other people won't have any idea where it is and will go: wtf. where is this from. Why not keep it within the file?

Reply:
You haven't resolve this.